inference-engine/tests/ngraph_functions/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ list(APPEND EXPORT_DEPENDENCIES
         ngraph_backend
         interpreter_backend
         inference_engine_lp_transformations
+        inference_engine_legacy
```

Comment:
why do we need legacy? ngraph function must not depend on legacy stuff, is not it?

Reply:
In general, in `ngraph function` shouldn't be depended on `inference_engine` and test utilities libs. But `LPT` part is depending on the both this libs (`inference_engine_lp_transformations` & ` inference_engine_legacy`). @eshoguli Could you comment this dependency? I think it will be better to separate LPT helpers to the stand-alone lib

Reply:
inference_engine_lp_transformations does not depend on legacy.

Reply:
Some helpers in this lib depend on it

Reply:
Was discussed offline